extensions/wikia/AdEngine/js/video/player/porvata/googleImaSetup.js

```diff
@@ -31,9 +26,8 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaSetup', [
 		var vastUrlBuilderOptions = {},
 			vastUrl;
 
-		if (megaIsEnabled(params)) {
-			vastUrlBuilderOptions.adUnit =
-				megaAdUnitBuilder.build(getPosBasedOnProduct(params), params.vastTargeting.src);
+		if (params.vastTargeting && params.vastTargeting.src && (params.adProduct || params.vastTargeting.pos)) {
```

Comment:
Do we need this if statement? How did it work before?

Reply:
Porvata2 will disappear, so I didn't want to dive more into it. Unit tests pass with it ;)